src/esp/assets/GenericMeshData.h

```diff
@@ -95,7 +95,7 @@ class GenericMeshData : public BaseMesh {
    * @ref renderingBuffer_.
    * @return Pointer to the compiled render mesh data.
    */
-  virtual Magnum::GL::Mesh* getMagnumGLMesh() override;
+  Magnum::GL::Mesh* getMagnumGLMesh() override;
```

Comment:
I personally like `virtual ... override`.  The `virtual` is redundant info because it's implied by `override`, which is likely why clang-tidy is removing it, but I think it makes sense that way you just have to look in one spot to see if a method is virtual or not.  I don't feel super strongly about this, but wanted to comment on why that is being done.

Reply:
Unfortunately, Clang-Tidy doesn't give us a lot of choice in the matter: https://clang.llvm.org/extra/clang-tidy/checks/modernize-use-override.html . If it was as simple as checking a config flag I would. I'd much rather have clang-tidy automatically add overrides than keep virtual though, since it actually helps ensure that we are annotating overrides correctly.

Reply:
Agreed, I really like override and would rather that be enforced.

Reply:
I'm a minimalist so if something can be expressed by just a single keyword instead of two, I prefer that. Same for `constexpr inline`, for example.